app/views/compute_resources_vms/form/vmware/_base.html.erb

```diff
@@ -49,6 +49,16 @@ end %>
                { :disabled => images.empty?, :label => _('Image'), :label_size => "col-md-2" } %>
 </div>
 
+<%= checkbox_f f, :secure_boot, { :help_inline => _("Enable Secure Bott for provisioning."),
```

Comment:
I think we should disable this option if the firmware is set to BIOS (as we did for Libvirt).